app/services/map_service.rb

```diff
@@ -0,0 +1,16 @@
+class MapService
+
+  def self.conn
+    Faraday.new(url: "http://www.mapquestapi.com")
+  end
+
+  def self.get_url(url)
+    response = conn.get(url)
+    JSON.parse(response.body, symbolize_names: true)
+  end
```

Comment:
Since this method is the same in both this `MapService` class and the `StationService` class, you could create a `BaseClass` that each service inherits from and move the method there. It would be a little more DRY.